packages/storybook/stories/ParticipantItem/snippets/CustomAvatar.snippet.tsx

```diff
@@ -1,19 +1,14 @@
 import { ParticipantItem } from '@azure/communication-react';
-import { PersonaPresence } from '@fluentui/react';
+import { Persona, PersonaPresence, PersonaSize } from '@fluentui/react';
 import React from 'react';
 
 export const CustomAvatarExample: () => JSX.Element = () => {
-  const avatarStyle = {
-    borderRadius: 20,
-    display: 'block'
-  };
   const onRenderAvatar = (): JSX.Element => {
     return (
-      <img
-        src="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/persona-female.png"
-        width="32px"
-        height="32px"
-        style={avatarStyle}
+      <Persona
+        size={PersonaSize.size32}
+        text="Annie Lindqvist"
```

Comment:
The participant item takes in a `displayName` so I would assume the avatar should just be the image and _not_ be the image + participantName. (In which case we have to make a significant change the participantItem component to not render the fluent persona text but instead have our own `<Text>{displayName)</Text>` item.
I'm not sure what regressions that would cause however so makes it a more risky fix.

Reply:
In the code for participant item, display name is only used here 
<img width="835" alt="Screen Shot 2022-04-29 at 4 30 05 PM" src="https://user-images.githubusercontent.com/96077406/166080582-af4428ab-0339-4ad2-b8f3-0e6c5bc871c5.png">
So in this case with onrenderavatar, it needs to pass in a <persona> not an image

Reply:
Updated the code to use options.text in onrenderavatar instead of directly passing in display name. This matches with how we use onrenderavatar elsewhere in the code space. :)